### PR TITLE
Various updates.

### PR DESCRIPTION
**Pull request by olafurpg**
- Add scripted tests.
- Inject resolver on typesafe-releases to resolve
  org.scala-sbt:interface:0.13.14-RC in builds using another sbt version.
- Add dottyLatestNightlyBuild helper to make it easier to use the latest
  Dotty version.

### CODE REVIEW

**Comment by olafurpg** [comment]
Review @felixmulder

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+pipeline:
+  test:
+    image: lampepfl/dotty:latest
+    pull: true
+    commands:
+      - sbt scripted
```
Comment (felixmulder) [inline .drone.yml]: What is the drone yaml file needed for?

**Review by felixmulder — COMMENTED** [review]
Otherwise LGTM

File: .drone.yml (hunk 1 above)
Comment (olafurpg) [reply .drone.yml]: Good catch! Removed.

**Pull request merged**